alts/src/main/java/io/grpc/alts/ComputeEngineChannelBuilder.java

```diff
@@ -26,7 +26,7 @@
  * using ALTS if applicable and using TLS as fallback.
  */
 public final class ComputeEngineChannelBuilder
-    extends ForwardingChannelBuilder<ComputeEngineChannelBuilder> {
+    extends ForwardingChannelBuilder2<ComputeEngineChannelBuilder> {
```

Comment:
This breaks ABI. We can't change this. Ditto for most of the usages in public APIs. AltsChannelBuilder has few enough users it is probably fine. BinderChannelBuilder would be nice to change, but we may not be easily able to. I'm actually surprised it isn't stable, but I guess it was just BinderServerBuilder that was marked stable.

Reply:
What about AndroidChannelBuilder?  That also looks to be stabilized.

Are we going to use AltsChannelBuilder2 & AndroidChannelBuilder2?

Reply:
Undid:

- AndroidChannelBuilder
- BinderChannelBuilder
- ComputeEngineChannelBuilder
- GoogleDefaultChannelBuilder